config.go

```diff
@@ -32,16 +32,18 @@ type menderConfig struct {
 		Key         string
 		SkipVerify  bool
 	}
-	RootfsPartA                  string
-	RootfsPartB                  string
-	UpdatePollIntervalSeconds    int
-	InventoryPollIntervalSeconds int
-	RetryPollIntervalSeconds     int
-	StateScriptTimeoutSeconds    int
-	ServerURL                    string
-	ServerCertificate            string
-	UpdateLogPath                string
-	TenantToken                  string
+	RootfsPartA                     string
+	RootfsPartB                     string
+	UpdatePollIntervalSeconds       int
+	InventoryPollIntervalSeconds    int
+	RetryPollIntervalSeconds        int
+	StateScriptTimeoutSeconds       int
+	StateScriptRetryTimeoutSeconds  int
+	StateScriptRetryIntervalSeconds int
```

Comment:
@pasinskim as far as I can see, the StatescriptTimeoutSeconds is the default timeout for a state-script. I thought the retry-scripts were supposed to have a default timeout?